plugins/env/app/views/samson_env/_environment_variables.html.erb

```diff
@@ -1,4 +1,4 @@
-<% scopes = [["All", nil]] + Environment.all.map{ |dg| [dg.name, "Environment-#{dg.id}"] } + [["----", nil]] + DeployGroup.all.map{ |dg| [dg.name, "DeployGroup-#{dg.id}"] } %>
+<% scopes = EnvironmentVariable.env_deploygroup_array %>
```

Comment:
guess this could be `[["All", nil]]` + xxx to avoid the conditional ... but either way is fine